### PR TITLE
Set the braintrust proxy as the api url to facilitate caching

### PR DESCRIPTION
**Pull request by ankrgyl**

### CODE REVIEW

File: py/autoevals/oai.py [hunk 1]
```diff
@@ -95,79 +76,39 @@ def post_process_response(resp):
...
-            output=resp["choices"],
-        )
-
-
-def run_cached_request(api_key=None, **kwargs):
+def run_cached_request(api_key=None, base_url=None, **kwargs):
     # OpenAI is very slow to import, so we only do it if we need it
-    complete, RateLimitError = prepare_openai_complete(is_async=False, api_key=api_key)
+    complete, RateLimitError = prepare_openai_complete(is_async=False, api_key=api_key, base_url=base_url)
+    print(kwargs)
```
Comment (manugoyal) [inline py/autoevals/oai.py:82]: remove?

File: py/autoevals/oai.py [hunk 2]
```diff
@@ -95,79 +76,39 @@ def post_process_response(resp):
...
+            retries += 1
 
     return resp
 
 
-async def arun_cached_request(api_key=None, **kwargs):
-    complete, RateLimitError = prepare_openai_complete(is_async=True, api_key=api_key)
+async def arun_cached_request(api_key=None, base_url=None, **kwargs):
+    complete, RateLimitError = prepare_openai_complete(is_async=True, api_key=api_key, base_url=base_url)
+    print(kwargs)
```
Comment (manugoyal) [inline py/autoevals/oai.py:100]: here too

**Review by manugoyal — APPROVED** [review]
exciting

File: py/autoevals/oai.py (hunk 1 above)
Comment (ankrgyl) [reply py/autoevals/oai.py:82]: 🤦

File: py/autoevals/oai.py (hunk 2 above)
Comment (ankrgyl) [reply py/autoevals/oai.py:100]: 🤦

**Pull request merged**